src/_pytest/monkeypatch.py

```diff
@@ -271,6 +271,19 @@ def syspath_prepend(self, path):
         # https://github.com/pypa/setuptools/blob/d8b901bc/docs/pkg_resources.txt#L162-L171
         fixup_namespace_packages(str(path))
 
+        # A call to syspathinsert() usually means that the caller wants to
+        # import some dynamically created files, thus with python3 we
+        # invalidate its import caches.
+        # This is especially important when any namespace package is in used,
+        # since then the mtime based FileFinder cache (that gets created in
+        # this case already) gets not invalidated when writing the new files
+        # quickly afterwards.
```

Comment:
```suggestion
        # quickly afterwards. See pallets/flask#3151 for reference.
```

I think it is worth adding a reference to the issue; although one can use "blame" to see the commit originally, it is very easy for this information to be lost by a rename or an auto-formatter (not really lost, but much harder to find). Adding a comment is a pretty painless way to avoid that IMHO. 👍

Reply:
It can also be found via the changelog (and this PR).. isn't that one of the reasons to have them? :)
I'd not add it, but feel free to do so, and merge it then.